### PR TITLE
desktop: Bump version to 6.7.0

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -955,15 +955,18 @@ jobs:
           paths:
             - desktop\release
 
+  # Publish all the artifacts generated by the previous wp-desktop- jobs to a
+  # GitHub release and generate the release notes using the Git commits history
   wp-desktop-publish:
     docker:
       - image: circleci/golang:1.12
-    working_directory: /home/circleci/wp-calypso
```
Comment (nsakaimbo) [inline .circleci/config.yml:961]: I'm surprised changing the path from `/home/circleci` to `~` is necessary to get this working.

For reference, [here](https://github.com/Automattic/wp-desktop/blob/2fbfbe63e834b6954a891a3bc5ab7686ba6171b5/.circleci/config.yml#L570-L613) is the identical job in the legacy wp-desktop repository.

File: .circleci/config.yml [hunk 2]
```diff
@@ -973,7 +976,7 @@ jobs:
             echo "Publishing draft release for wp-desktop $VERSION..."
             NAME="WP-Desktop ${VERSION#?}"
 
-            chmod +x desktop/bin/make-changelog.sh
+            chmod +x ./desktop/bin/make-changelog.sh
```
Comment (nsakaimbo) [inline .circleci/config.yml]: 🤔  This seems kinda strange, no? `chmod +x path/to/file` worked fine in the legacy repo, not sure why the `.` prefix is necessary here?

File: .circleci/config.yml (hunk 1 above)
Comment (mokagio) [reply .circleci/config.yml:961]: You are right. Turns out the only thing I needed was to add the `checkout` step _before_ the `attach_workspace: at:` one. See https://github.com/Automattic/wp-calypso/pull/48331.

It just took me a few experiments to figure it out and this branch still carried most of them.

I reverted to the minimum required set of changes now.

Comment (nsakaimbo) [reply .circleci/config.yml:961]: Ah - yeah, adding the checkout step makes sense (so the git history will be available). Thank you for catching this, and for fixing! 🙇

**Review by nsakaimbo — APPROVED** [review]
👍

**Pull request merged**